build-scripts/buildvm/docker_image_cleanup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+THREEWEEKSAGO="$(date -Ins --date="3 Weeks Ago")"
+
+docker images -q | while read IMAGE_ID; do
```

Comment:
Don't really need the line-reading capabilities of `while read` here, could just

```sh
for image in $( docker images -q ); do

done
```